content/docs/examples/multicluster/split-horizon-eds/index.md

```diff
@@ -84,7 +84,7 @@ This will be used to access pilot on `cluster1` securely using the ingress gatew
     Wait for the Istio pods on `cluster1` to become ready:
 
     {{< text bash >}}
-    $ kubectl get pods --context=$CTX_CLUSTER1 -n istio-system
+    $ kubectl get pods --context=$CTX_CLUSTER1 -n istio-system --field-selector=status.phase!=Succeeded
```

Comment:
I'm wondering if this is really needed here. Isn't it better to let the user see all the pods, whether they've done something during startup and completed, or are up and running?

Reply:
I was trying to get the command to generate the output that was included  in the text bash which had no `Completed` pods. I'm fine going back to the old commands, updating the sample output to include the `Completed` pods, and possibly adding a remark the all pods should be `Running` or `Completed`. Thoughts @frankbu on which makes sense.

Reply:
Yeah, I don't know. I notice that every time you update istio, you get another set of these post-install and cleanup `Completed` pods, so the output from `get po` keeps getting more and more messy. I think we need to look into a way to delete these pods, rather than deal with them in the docs.

Reply:
I agree with the messiness of the `k get po` output. Trying to generate an acceptable command string can be difficult and error prone. This is where having something in `istioctl` to verify readiness would be a good thing.

We could do something about removing the pods, but then I wonder if we would want people to be able to gather logs for diagnosis of a problem and then they can't.

Reply:
Discussion with @frankbu: We will leave the commands to show the `Completed` pods and add the `Completed` pods to the sample output for consistency.

Reply:
Commit # 3 has this work.